src/docs/PML/PMLUserGuide.adoc

```diff
@@ -2898,35 +2880,68 @@ NOTE: Your CSV file should be in a table format. See the example below.
 
 image::csv_file_organisation.png[align="center"]
 
-===== Import_Data_Interactive
+===== Import_Data_And_Automate_Feature_Engineering
 
-*Task Overview:* Load data from external sources, predict its features types and assist data scientists to easily encode categorical data.
+*Task Overview:* This workflow provides a complete solution to assist data scientists to successfully load and encode their categorical data.
+It currently supports different encoding methods such as Label, OneHot, Dummy, Binary, Base N, Hash and Target.
+It also enables:
 
-*Task Variables:*
+- Automatic identification of the best-suited method for encoding each categorical column, when no encoding method is selected (Auto mode).
+- Data type recognition: identification of the data type of each column (categorical or numerical).
+- Creation of summary statistics for each column: missing values, minimum, maximum, average, zeros, and cardinality.
+- Editing of the data structure: modification of column information (name, type, category, etc.), deletion of a column, etc.
+
+This workflow can be used:
+
+- Stand-alone such that the results can be saved in the User Data Space or locally.
+- In a larger workflow where the results will be sent to the next connected task.
+
+NOTE: For further information, please check the subsection <<AutoFeat>>.
 
-.Import_Data_Interactive_Task variables
+
+.Import_Data_And_Automate_Feature_Engineering variables
 [cols="2,5,2"]
 |===
 | *Variable name* | *Description* | *Type*
-| `TASK_ENABLED`
-| If False, the task will be ignored, it will not be executed.
-| Boolean (default=True)
+3+^|*Workflow variables*
 | `IMPORT_FROM`
-| Selects the type of data source.
+| Selects the method/protocol to import the data source.
 | List [PA:URL,PA:URI,PA:USER_FILE,PA:GLOBAL_FILE] (default=PA:URL)
 | `FILE_PATH`
-| Inserts a file path/name.
+| Inserts the path/name of the file that contains the dataset.
 | String
 | `FILE_DELIMITER`
 | Defines a delimiter to use.
 | String (default=;)
 | `LIMIT_OUTPUT_VIEW`
-|  Specifies how many rows of the dataframe will be previewed in the browser to check each task results.
+|  Specifies how many rows of the dataframe will be previewed in the browser to check the encoding results.
 | Int (-1 means preview all the rows)
+| `NATIVE_SCHEDULER`
+| Name of the Native Scheduler node source to use when the workflow tasks must be deployed inside a cluster such as SLURM, LSF, etc.
+| String (default=empty)
+| `NATIVE_SCHEDULER_PARAMS`
+|Parameters given to the native scheduler (SLURM, LSF, etc) while requesting a ProActive node used to deploy the workflow tasks.
```

Comment:
ProActive node used to deploy the workflow tasks. -> ProActive node used to execute the workflow tasks.